addons/website_event_track/models/event_track.py

```diff
@@ -630,3 +648,91 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
         )
 
         return track_candidates[:limit]
+
+    def _get_reminder_times_warning(self):
+        return Markup('<strong><u>%(warning_title)s</u></strong>: %(warning_content)s') % {
+            'warning_title': _('Note'),
+            'warning_content': _(
+                'The start and end times of the talk were not specified when you asked to add them to your calendar, '
+                'therefore the times indicated in this reminder correspond to those of the event.'
+            )
+        } if not self.date else ''
+
+    def _get_external_description(self):
+        """ Adding the URL of the event track into the description """
+        self.ensure_one()
+        return Markup("<a href='%(event_track_url)s'>%(name)s</a>\n%(shorten_description)s\n\n%(reminder_times_warning)s") % {
+            'event_track_url': werkzeug.urls.url_join(self.get_base_url(), self.website_url),
+            'name': self.name,
+            'shorten_description': shorten(html_to_inner_content(self.description), 1900),
+            'reminder_times_warning': self._get_reminder_times_warning(),
+        }
+
+
+    def _get_event_track_reminder_dates(self):
+        return {
+            'date_begin': self.date or self.event_id.date_begin,
```

Comment:
It is a bit weird... but ok. Or shouldn't we prevent adding in calendar tracks that have no dates ? Because it won't make any sense to have a track for the whole 4 days, in the end you don't have any valuable information in your calendar. Could be a check done at controller level: if trying to send a reminder / get ics file for tracks without date, just raise / return an error. Moreover normally published tracks should have dates, so makes more sense than sending fake dates. What do you think ?